.github/workflows/ci-owasp-dep-check.yaml

```diff
@@ -0,0 +1,94 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: CI - Misc - OWASP Dependency Check
+on:
+  pull_request:
```

Comment:
Why running this check on every pull request?
New dependencies are added very seldom.
If a dependency is flagged there will be much noise.
People usually add a new dependency using the latest available version.

I suggest to run this only periodically, like every day and manually on every active release branch

Reply:
@eolivelli this is only run when one of the pom.xml files is modified.

Reply:
in addition there's the existing job to run periodically

Reply:
@eolivelli as Lari said, it runs only on changes in pom files, see:
```
      - name: Detect changed pom files
        id: changes
        uses: apache/pulsar-test-infra/paths-filter@master
        with:
          filters: |
            poms:
              - 'pom.xml'
              - '**/pom.xml'
```